cosmetics-web/spec/support/feature_helpers.rb

```diff
@@ -529,3 +529,77 @@ def answer_does_product_contain_poisonous_ingredients_with(answer)
 def add_an_item
   click_button "Add item"
 end
+
+
+
+#Create responsible person helpers
+
+def go_to_rp_declaration_page
+    visit(root_path)
+    expect(page).to have_h1("Responsible Person Declaration")
+    click_button 'I confirm'
+    expect(page).to have_h1("Are you or your organisation a UK Responsible Person?")
+    click_on 'Continue'
+    expect(page).to have_h1("Does anyone in your organisation have an account to submit cosmetic product notifications in the UK?")
+     choose "option_create_new", visible: false
+     click_on "Continue"
+    expect(page).to have_h1("Is the UK Responsible Person a business or an individual?")
+end
+
+
+
+
+def create_individual_responsible_person
+    go_to_rp_declaration_page
+    select_individual_account_type
+
+    assert_text "UK Responsible Person details"
+
+    fill_in "Name", with: responsible_person.name
+    fill_in "Building and street", with: responsible_person.address_line_1
+    fill_in "Town or city", with: responsible_person.city
+    fill_in "County", with: responsible_person.county
+    fill_in "Postcode", with: responsible_person.postal_code
+    click_on "Continue"
+
+    assert_text "contact person"
+
+    fill_in "Full name", with: responsible_person.contact_persons.first.name
+    fill_in "Email address", with: responsible_person.contact_persons.first.email_address
+    fill_in "Phone number", with: responsible_person.contact_persons.first.phone_number
+    click_on "Continue"
+  end
+
+  def create_business_responsible_person
+    go_to_rp_declaration_page
+    select_business_account_type
+
+    assert_text "UK Responsible Person details"
+
+    fill_in "Business name", with: business_responsible_person.name
+    fill_in "Building and street", with: business_responsible_person.address_line_1
+    fill_in "Town or city", with: business_responsible_person.city
+    fill_in "County", with: business_responsible_person.county
+    fill_in "Postcode", with: business_responsible_person.postal_code
+    click_on "Continue"
+
+    assert_text "contact person"
+
+    fill_in "Full name", with: business_responsible_person.contact_persons.first.name
+    fill_in "Email address", with: business_responsible_person.contact_persons.first.email_address
+    fill_in "Phone number", with: business_responsible_person.contact_persons.first.phone_number
+    click_on "Continue"
+  end
+
+
+  def select_business_account_type
+    assert_text "Is the UK Responsible Person a business or an individual?"
+    choose "responsible_person_account_type_business", visible: false
```

Comment:
I think ideally we should use the visible text rather than `name` attribute of the `<input>`, eg:

```suggestion
    choose "Limited company or Limited Liability Partnership (LLP)"
```

...also I think it should be visible?